docs/usage.md

```diff
@@ -24,29 +25,63 @@ At minimum, four parameters are required:
 3. `--refphylogeny`: Reference phylogeny.
 4. `--model`: Evolutionary model used when estimating the phylogeny, e.g. "LG+F+R6".
 
-## Samplesheet input
+## Samplesheet input for phylogenetic placement
 
 Each of the four parameters mentioned above can be specified as columns in a comma separated sample sheet instead.
 In addition, a `sample` column needs to be present.
 
 ```bash
---input '[path to samplesheet file]'
+--phyloplace_input '[path to samplesheet file]'
 ```
 
-```console
+```csv title="phyloplace_sheet.csv"
 sample,queryseqfile,refseqfile,refphylogeny,model
 pp0,q0.faa,ref0.alnfaa,ref0.newick,LG
 pp1,q1.faa,ref1.alnfaa,ref1.newick,LG+F+R6
 ```
 
+## Samplesheet input for search followed by phylogenetic placement
+
+This mode of the pipeline starts by searching a fasta file with a set of HMMER `hmm` files.
+The results are then passed to phylogenetic placement.
+The samplesheet for this mode hence needs to contain paths to `hmm` files plus the phylogenetic placement information.
+(_Note_: If an `hmm` file is not accompanied by a reference tree, plus the associated information, this will be used to search, but not phylogenetic placement, and the sequences will appear in a result table.
+In the below example, the `rnr` entry will only be used for searching, while the other two will be both searched for and placed.)
```

Comment:
```suggestion

:::note
If an `hmm` file is not accompanied by a reference tree, plus the associated information, this will be used to search, but not phylogenetic placement, and the sequences will appear in a result table.
In the below example, the `rnr` entry will only be used for searching, while the other two will be both searched for and placed.)
:::
see https://nf-co.re/docs/contributing/website/markdown for the syntax
```